docs-src/ref.scrbl

```diff
@@ -268,6 +273,32 @@ In either case, the key will be displayed to the console upon start-up.
 
 ]
 
+@subsection[#:tag "ref-usage-rpc-dev"]{@tt{reach rpc-dev}}
+
+The sub-command
+
+@cmd{reach rpc-dev CMD}
```

Comment:
If this command is necessary, then I think a better name might be something that includes the word `run`, like `run-rpc`, and that starts up the server itself behind the scenes.

Reply:
Resolved in 0289e8495254ef2b2a1f33c567422bc1fade9a57.